index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width , initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Tayyab Kamboh Personal Portfolio</title>
+</head>
+<body>
+    <div class="container"> 
+        <div class="headline-container">
+        
+    </div>   
+
+    <nav class="toolbar-container">
+        <div class="name-container">
+         <a> Tayyab</a>  
+            
+        </div>
+        <div class="hamburger">
+            <span class="bar line-1"></span>
+            <span class="bar line-2"></span>
+            <span class="bar line-3"></span>                    
+        </div> 
+    </nav>
```

Comment:
- [ ] Kindly replace `<nav>` with `<header>` tag and add replace the div tag with `<nav>` tag for the hamburger icon to improve SEO for your portfolio.